index.md

```diff
@@ -230,6 +230,30 @@ The example below applies a `margin-right` style to the first input.
   <<< .vitepress/docs/components/HorizontalForm.vue
 </SplitTab>
 
+### Prop: schemaRowClasses
+
+If you ever require to add additional classes to each structural row in the generated form, you can use the `schemaRowClasses` property of the `SchemaForm` component to inject them.
+
+These classes will be appended, and will _not_ substitute the `schema-row` class that FormVueLate already provides.
+
+The `schemaRowClasses` prop accepts `[String, Object, Array]` as valid types, conforming to Vue's [class and style binding syntax](https://v3.vuejs.org/guide/class-and-style.html#binding-html-classes)
+
+```html
+<SchemaForm
+  schemaRowClasses="my-custom-class-a my-custom-class-b"
+  :schema="mySchema"
+/>
+```
+
+Example output:
+
+```html
+<form>
+  <div class="schema-row custom-class-a custom-class-b">
+    [...]
+  </div>
+</form>
+```
```

Comment:
Could you please also add a space after this bit?

Reply:
Done. I love your attention to this details. Have to improve personally for this.